content/_index.md

```diff
@@ -6,4 +6,13 @@ title: "Runbooks"
 
 - We are tired of haphazardly hunting through messy threads of GitHub issues and StackOverflow when we hit a problem
 - We don't want a one-off fix, we want to deepen our understanding of the problem space
+- We want to give people a resource where they can benefit from our experience without taking up our time
```

Comment:
I think I get how you mean this, but I believe it could be read in the wrong way. Maybe:
`We want to give people a resource where they can benefit from our experiences and allows us to focus on other challenges instead of the same reoccurring problems`